core/embed/rust/src/ui/layout_eckhart/firmware/device_menu_screen.rs

```diff
@@ -269,6 +268,8 @@ enum Subscreen {
     AboutScreen,
     // A screen showing the regulatory information
     RegulatoryScreen,
+    // A screen showing information about a backup
```

Comment:
Nit: not about "a backup". It's literally *the* backup.

That being said, the comment doesn't add much value as the enum member explains it all.